services/gundeck/src/Gundeck/Aws.hs

```diff
@@ -346,8 +346,8 @@ timeToLive t s = Attributes (Endo (ttlAttr s))
     ttlNow GCM             = "0"
     ttlNow APNS            = "0"
     ttlNow APNSSandbox     = "0"
-    ttlNow APNSVoIP        = "15" -- See note [VoIP TTLs]
-    ttlNow APNSVoIPSandbox = "15" -- See note [VoIP TTLs]
+    ttlNow APNSVoIP        = "45" -- See note [VoIP TTLs]
+    ttlNow APNSVoIPSandbox = "45" -- See note [VoIP TTLs]
```

Comment:
In the absence of fallbacks, would it potentially make sense to A) further increase this TTL (45s for APNS / 4weeks for GCM (default) seems odd) and B) make it configurable?